src/table/table-wrapper.tsx

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+
+export interface TableWrapperProps {
+  children: React.ReactNode;
+  className: string;
```

Comment:
Would it make more sense to pass `hasFooter` and `hasHeader` and define the class name inside?